silo/views.py

```diff
@@ -944,15 +951,24 @@ def list_silos(request):
     Each silo is listed with links to details
     """
     user = User.objects.get(username__exact=request.user)
+    user_wfl1s = get_workflowlevel1s(user)
 
     # get all of the silos
     own_silos = Silo.objects.filter(owner=user).prefetch_related('reads')
 
-    shared_silos = Silo.objects.filter(Q(shared__id=user.pk) |
-                                       Q(share_with_organization=True,
-                                         owner__tola_user__organization=\
-                                         user.tola_user.organization))\
-        .exclude(owner=user).prefetch_related("reads")
+    if user_wfl1s:
```

Comment:
The same thing that I mentioned in the `edit_silo` function. I don't think you need this conditional.